docs/creating-extensions.md

```diff
@@ -150,8 +152,10 @@ Use `%env(VAR_NAME)%` syntax in service configuration to reference environment v
 - `exclude`: Exclude specific class names (string or array)
 - `include_only`: Only load specific class names (string or array)
 - Cannot use both `exclude` and `include_only` for same package
+FIXME: Should we out source this to the extension instead? Ie, let them provide config to disable?
```

Comment:
hmm i also not absolutly like the notation by now. let us create an issue for that